src/System.Runtime.InteropServices.RuntimeInformation/src/Architecture.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public enum Architecture
```

Comment:
To separate the interop layer with the lib code. We may reuse GetUnixArchitecture() elsewhere without having to reference RuntimeInformation for Architecture.